src/pages/Header.jsx

```diff
@@ -46,13 +46,10 @@ const Header = function() {
                 Support<span className="HideOnSmallScreen"> the Daily</span>
               </button>
               <select name="amount" className="tsd-select form-control">
-                <option value={5}>$5</option>
-                <option value={10}>$10</option>
-                <option value={25}>$25</option>
                 <option value={50}>$50</option>
-                <option value={100}>$100</option>
-                <option value={500}>$500</option>
```

Comment:
Can you set the value of a3 to also be 50 and not 5? (later in this page) (see https://stackoverflow.com/questions/11019954/paypal-recurring-payments-form)